python/lib/dependabot/python/update_checker/pip_compile_version_resolver.rb

```diff
@@ -251,7 +251,17 @@ def pip_compile_options(filename)
             options << "--output-file=#{requirements_file.name}"
           end
 
-          options.join(" ")
+          if (requirements_file = compiled_file_for_filename(filename))
+            if requirements_file.content.include?("uv pip compile")
+              options += uv_pip_compile_options_from_compiled_file(requirements_file)
```

Comment:
Hi @avilaton , looks like you added 2 new functions `uv_pip_compile_options_from_compiled_file `, `pip_compile_options_from_compiled_file` in `pip_compile_version_resolver.rb`, i think they are being referenced from `pip_compile_file_updater.rb`  can you please fix this (IDE is showing it as no function exists in  `Dependabot::Python::UpdateChecker::PipCompileVersionResolver`)

Reply:
Would be neat to have a way not to repeat this code, it seems to me as if it would be more resilient to have them centralized. What do you suggest we should do?

Reply:
@avilaton , if you think we might benefit from moving them to new location/class and see a additional functionality that could be used across. please do so.